appveyor.yml

```diff
@@ -1,7 +1,5 @@
 environment:
   matrix:
-  - julia_version: 0.6
```

Comment:
Same here. As long as we still support 0.6 and 0.7, add a comment explaining why they're disabled.